tools/autodeployment/src/sibling-checker/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-expressions */
+const fetch = require('node-fetch');
+const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
+
+const server =
+  process.DEPLOY_TYPE === 'production'
+    ? 'https://telescope.cdot.systems/health.'
+    : 'https://dev.telescope.cdot.systems/health.';
+
+let timerOne;
+let timerTwo;
```

Comment:
These could be `healthTimer` and `whateverNameYouHaveForThisTimer`.

Reply:
`timerOne` and `timerTwo` doesn't tell you anything about what their purpose is or why they exist.